src/helpers/postFeedHelpers.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
```

Comment:
It's better to keep this file in the related directory. If it's a helper that is going to be used only for PostFeedCard it's better to be named `PostFeedCardHelper.js` and be saved alongside the component. This folder better to be kept only for general helpers.